samples/dependency-injection/aspnetcore/Core_7.2/Sample/Sample.Core.csproj

```diff
@@ -7,6 +7,6 @@
     <ProjectReference Include="..\NServiceBus.Extensions.DependencyInjection\NServiceBus.Extensions.DependencyInjection.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore" Version="2.*" />
```

Comment:
Should we not pin these, to get repeatable builds?

Reply:
All our samples seem to do this so I didn't want to change. Perhaps raise an issue to discuss?